File: webapp/HIN_WEB/ClientApp/src/app/model/Country.ts
export class Country {
  public Id: number;
  public SortName: string;
  public Name: string;
  public PhoneCode: number;
}

export class State {
  public Id: number;
  public Name: string;
  public CountryId: number;
}

export class City {
  public Id: number;
  public Name: string;
  public StateId: number;
}

File: api/HIN_API/Models/Country.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HIN_API.Models
{
    public partial class Country
    {
        [Key]
        public int Id { get; set; }
        [Required]
        [StringLength(5)]
        public string SortName { get; set; }
        [Required]
        public string Name { get; set; }
        public int PhoneCode { get; set; }
    }
}
